samples/camera/camera.rst

```diff
@@ -0,0 +1,10 @@
+.. _camera-samples:
```

Comment:
You'll also need to add a mention of this ``camera.rst`` document in the ``samples/index.rst`` file.  (That will fix the ``WARNING: document isn't included in any toctree`` messages.)

Reply:
Remove the camera.rst

Reply:
Yes, agreed to just have a readme.rst makes sense, but you'll still need to add this readme.rst to the samples/index.rst to have it included in the doc tree.  Also @JunYangNXP @loicpoulain it seems some synergy possible looking at this PR and the Video API, Drivers and Sample #17194 PR.

Reply:
@dbkinder Added to the samples/index.rst and passed the Doc review.
@loicpoulain I will look into your PR.